src/interpretesencillo/InterpreteSencillo.java

```diff
@@ -56,12 +56,18 @@ private static void ejecutarAST(LinkedList<Instruccion> ast) {
         TablaDeSimbolos ts=new TablaDeSimbolos();
         //Se ejecuta cada instruccion en el ast, es decir, cada instruccion de 
         //la lista principal de instrucciones.
+        String var;
         for(Instruccion ins:ast){
             //Si existe un error léxico o sintáctico en cierta instrucción esta
             //será inválida y se cargará como null, por lo tanto no deberá ejecutarse
-            //es por esto que se hace esta validación.
-            if(ins!=null)
-                ins.ejecutar(ts);
+            //es por esto que se hace esta validación.            
+            try{
+                if(ins!=null)
+                    ins.ejecutar(ts);
+            }
+            catch(NullPointerException e){
```

Comment:
@gl3ncal3l te comento que NullPointerException es una [Unchecked Exception](https://crunchify.com/better-understanding-on-checked-vs-unchecked-exceptions-how-to-handle-exception-better-way-in-java/) y como buena practica no se deben capturar con la sentencia Try-Cath, en su lugar se deben validar.

Como podes ver, `ins` ya se valida que no sea null antes de ejecutar,  lo ideal sería agregar un sentencia else (en el caso de que sea null) y ejecutar alli tus sentencias.

Por otro lado, cual es la razón de hacer un break? la ideal de la valiación de null con el if es evitar que la ejecución se detenga, con el break se detendría. Me podes explicar por favor?

Saludos.